logical_switch_port.go

```diff
@@ -331,7 +331,6 @@ func (odbi *ovndb) rowToLogicalPort(uuid string) *LogicalSwitchPort {
 		default:
 			//	glog.V(OVNLOGLEVEL).Info("Unsupport type found in lport dynamic address.")
```

Comment:
Can you also get rid of   unwanted comment //	glog.V(OVNLOGLEVEL) and return error for unsupported type instead
return nil, fmt.Errorf("Unsupport type found in lport dynamic address.")
. Seems we missed this in previous review.